server/block/jukebox.go

```diff
@@ -0,0 +1,112 @@
+package block
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/internal/nbtconv"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/sound"
+	"time"
+)
+
+// Jukebox is a block used to play music discs.
+type Jukebox struct {
+	solid
+	bass
+
+	// Item is the music disc played by the jukebox.
+	Item item.Stack
+}
+
+// FuelInfo ...
+func (j Jukebox) FuelInfo() item.FuelInfo {
+	return newFuelInfo(time.Second * 15)
+}
+
+// BreakInfo ...
+func (j Jukebox) BreakInfo() BreakInfo {
+	d := []item.Stack{item.NewStack(Jukebox{}, 1)}
+	if !j.Item.Empty() {
+		d = append(d, j.Item)
+	}
+	return newBreakInfo(2, alwaysHarvestable, axeEffective, simpleDrops(d...)).withBreakHandler(func(pos cube.Pos, w *world.World, u item.User) {
```

Comment:
```suggestion
	return newBreakInfo(0.8, alwaysHarvestable, axeEffective, simpleDrops(d...)).withBreakHandler(func(pos cube.Pos, w *world.World, u item.User) {
```

Reply:
where did you get this information from?

Reply:
Resolved on discord